inloop/grading/models.py

```diff
@@ -8,7 +9,29 @@ class PlagiarismTest(models.Model):
     command = models.TextField(default="", help_text="Command that was used to perform the test")
 
     def __str__(self):
-        return self.command
+        return "Plagiarism test #{}".format(self.id)
+
+    @property
+    def all_detected_plagiarisms(self):
+        """
+        Get all detected plagiarisms to this Test.
+        """
+        return DetectedPlagiarism.objects.filter(test=self, veto=False)
+
+    @property
+    def time_description(self):
```

Comment:
As discussed in a meeting with @martinmo, time_description is already implemented by Django, so this function can be simplified.

Reply:
Exaclty, it is located here: https://github.com/django/django/blob/stable/1.11.x/django/utils/timesince.py